coderedcms/project_template/website/migrations/0001_initial.py

```diff
@@ -128,4 +128,9 @@ class Migration(migrations.Migration):
             },
             bases=('coderedcms.coderedpage',),
         ),
+        migrations.AddField(
+            model_name='formconfirmemail',
+            name='page',
+            field=modelcluster.fields.ParentalKey(on_delete=django.db.models.deletion.CASCADE, related_name='confirmation_emails', to='website.FormPage'),
+        ),
```

Comment:
Your branch should not have any changes to migrations... it should be exact same as master branch. Please figure out why (maybe due to merge) and make sure this PR does not affect migrations.